components/tyk-gateway/templates/service-gw-control.yaml

```diff
@@ -20,6 +20,7 @@ spec:
   - port: {{ .Values.gateway.control.port }}
     targetPort: {{ .Values.gateway.control.containerPort }}
     protocol: TCP
+    name: {{ default "http" .Values.gateway.control.portName | quote }}
```

Comment:
do we need to give default name for tyk-gateway service? Since it had no port name previously, maybe we can skip giving a default name